nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -62,7 +62,7 @@
 #include "NXvars.h"
 #include "NXproto.h"
 
-#include "xfixesproto.h"
+#include <X11/extensions/xfixesproto.h>
 #define Window     XlibWindow
 #define Atom   XlibAtom
 #define Time XlibXID
```

Comment:
`nx-X11/programs/Xserver/hw/nxagent/Composite.c` includes `composite.h`, which must also be changed in the same vain.